utils/fs_utils.go

```diff
@@ -6,12 +6,46 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"sort"
 	"strings"
 
 	"github.com/golang/glog"
 )
 
-func GetDirectorySize(path string) (int64, error) {
+// Directory stores a representaiton of a file directory.
+type Directory struct {
+	Root    string
+	Content []string
+}
+
+type DirectoryEntry struct {
+	Name string
+	Size int64
+}
+
+type EntryDiff struct {
+	Name  string
+	Size1 int64
+	Size2 int64
+}
+
+func GetSize(path string) int64 {
+	stat, err := os.Stat(path)
+	if err != nil {
+		glog.Errorf("Could not obtain size for %s: %s", path, err)
+		return -1
```

Comment:
seems to me that if we hit an error here, we have another problem and probably want to throw some kind of error, rather than just silently returning -1 (which will just get outputted silently). WDYT?